vertx-client/build.gradle

```diff
@@ -56,7 +56,11 @@ dependencies {
     api(group: 'io.vertx', name: 'vertx-web-client', version: '4.4.1') {
         exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'
     }
-    implementation(group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.44.Final')  // for openSSL
+    implementation(group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.61.Final')  // for openSSL
+    implementation(group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.61.Final', classifier: 'linux-x86_64')  // for openSSL
+    implementation(group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.61.Final', classifier: 'osx-aarch_64')  // for openSSL
+    implementation(group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.61.Final', classifier: 'linux-aarch_64')  // for openSSL
+    implementation(group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.61.Final', classifier: 'osx-x86_64')  // for openSSL
```

Comment:
Hmm... I think the dependency at line#59 already includes all variants. It either downloads all variants or download with a classifier.

Reply:
actually, behavior changed after version `2.0.48.Final`. https://github.com/netty/netty-tcnative/issues/716